### PR TITLE
add help message support for gfxmenu

### PR DESCRIPTION
**Pull request by sTeeLM**
bug fix 版本，build通过，测试通过

### CODE REVIEW

File: grub-core/commands/menuentry.c [hunk 1]
```diff
@@ -45,6 +45,7 @@ static const struct grub_arg_option options[] =
        anyone can boot it.  */
     {"unrestricted", 0, 0, N_("This entry can be booted by any user."),
      0, ARG_TYPE_NONE},
+    {"help", 0, GRUB_ARG_OPTION_OPTIONAL,  N_("Menu entry help message."), N_("STRING"), ARG_TYPE_STRING},
```
Comment (a1ive) [inline grub-core/commands/menuentry.c:48]: 要不这里用 help-msg 吧，别和原来的 help 冲突了
{"help-msg", ...

Comment (sTeeLM) [reply grub-core/commands/menuentry.c]: 嗯，可以，我重新提个pr？

Comment (a1ive) [reply grub-core/commands/menuentry.c]: 好。

**Comment by steve6375** [comment]
If not using a theme, does help_msg still work or is it ignored?
If it only works with a theme, then maybe call it --theme-help ?

**Pull request closed without merging**